### ISSUE TITLE
[YCQL] yugabyte db 2.2 crashes when running cassandra-migrate

### ISSUE THREAD

**Issue by suppandi** [yugabyte/yugabyte-db#5270]
Started yugabyte 2.2.0-b8 using the helm chart from yugabyte.

```
helm install yugabyte yugabyte \
   --set storage.ephemeral=true \
   --set replicas.master=1 \
   --set replicas.tserver=1 \
   --set storage.master.count=1 \
   --set storage.tserver.count=1 \
   --set enableLoadBalancer=false \
   --set resource.master.requests.memory=0.2Gi \
   --set resource.tserver.requests.memory=0.2Gi
```

ran [cassandra-migrate](https://github.com/Cobliteam/cassandra-migrate) with a simple config
```
cassandra-migrate -c ./config -m default -H $CASSANDRA_HOST -p $CASSANDRA_PORT migrate
```

config file is
```
$ cat config
keyspace: testg
profiles:
  default:
    replication:
      class: SimpleStrategy
      replication_factor: 1
migrations_path: ./db-migrations
new_migration_name: "v{next_version:03d}_{desc}"
```

db-migrations has
```
$ ls db-migrations/
v001_newbaseline.cql
$ cat db-migrations/v001_newbaseline.cql
create table  account(
    id text,
    email text,
    password blob,
    primary key (id, email)
)  WITH transactions = { 'enabled' : true };;
```

Attaching log files for tserver pod and cassandra-migrate.
[yb-tserver.log](https://github.com/yugabyte/yugabyte-db/files/5003185/yb-tserver.log)
[cassandra-migrate.log](https://github.com/yugabyte/yugabyte-db/files/5003189/cassandra-migrate.log)

**Comment by frozenspider** [comment]
Easy to reproduce just by doing
```cql
INSERT INTO k.t (content) VALUES ('
');
```
(line break in a string causes this)

For reference, the original query causing the crash was:
```cql
INSERT INTO "testg"."database_migrations"
(id, version, name, content, checksum, state, applied_at)
VALUES (0c8fc67d-a322-4487-acff-9020e900893d, 1, 'v001_newbaseline.cql', 'create table  account(
    id text,
    email text,
    password blob,
    primary key (id, email)
)  WITH transactions = { ''enabled'' : true };

', 0xf2b9979dd16b3355baa796ded03df6f78ef23cfc2e462288da0021858cc3f025, 'IN_PROGRESS', toTimestamp(now())) IF NOT EXISTS
```

**Comment by nocaway** [comment]
This is fixed in master and backported to 2.2 branch
